ci-operator/step-registry/openshift/microshift/e2e/router-smoke-test/openshift-microshift-e2e-router-smoke-test-commands.sh

```diff
@@ -105,12 +105,49 @@ RESULT=$?
 echo "${RESPONSE}"
 
 if [ $RESULT -ne 0 ] || ! echo "${RESPONSE}" | grep -q -E "HTTP.*200 OK"; then
+  cat <<'EOF' > "${HOME}"/debug.sh
+#!/usr/bin/env bash
+set -x
+
+echo
+echo "=== DEBUG INFORMATION ==="
+echo
+
+oc get nodes
+oc get nodes -o yaml
+oc get pods -A
+oc get pods -A -o yaml
+oc get svc -A
+oc get svc -A -o yaml
+oc get routes -A
+oc get routes -A -o yaml
+oc get events -A
+
+for ns in $(oc get namespace -o jsonpath='{.items..metadata.name}'); do
+      for pod in $(oc get pods -n $ns -o name); do
+            oc describe -n $ns $pod || true
+            for container in $(oc get -n $ns $pod -o jsonpath='{.spec.containers[*].name}'); do
+                  oc logs -n $ns $pod $container
+                  oc logs --previous=true -n $ns $pod $container
+            done
+      done
+done
+EOF
```

Comment:
We have this code in 2 places already, don't we? Should we add this script under ./hack/ and reuse it in the CI?